### ISSUE TITLE
Resume Training Process

### ISSUE THREAD

**Issue by kartiksau89** [MDIL-SNU/SevenNet#226]
Hello, how to resume the training process once we stop?

**Comment by YutackPark** [comment]
Check this example: https://github.com/MDIL-SNU/SevenNet/blob/main/sevenn/presets/fine_tune.yaml

The key is:
```yaml
    continue:
        reset_optimizer: True
        reset_scheduler: True
        reset_epoch: True
        checkpoint: {checkpoint_xx.pth}
```

If you just want to "continue" training without changing optimizer or scheduler, put all `reset_` keys to `False`.

If you wish to change the optimizer or scheduler for fine-tuning or for some reason, you should put `reset_` keys to `True`. In this case, the initial epochs of training could be unstable.

**Comment by kartiksau89** [comment]
Thanks.
